manifests-maistra/2.3.0/maistraoperator.v2.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,794 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: maistraoperator.v2.3.0
+  namespace: placeholder
+  labels:
+    operatorframework.io/arch.amd64: supported
+
+  annotations:
+    categories: "OpenShift Optional, Integration & Delivery"
+    capabilities: "Seamless Upgrades"
+    certified: "false"
+    repository: https://github.com/maistra/istio-operator
+    description: |-
+      The Maistra Operator enables you to install, configure, and manage an instance of Maistra service mesh. Maistra is based on the open source Istio project.
+
+    containerImage: quay.io/maistra/istio-ubi8-operator:2.3.0
+    createdAt: 2022-08-30T16:21:51CEST
+    support: Red Hat, Inc. 
+    olm.skipRange: ">=1.0.2 <2.3.0-0"
+    operators.openshift.io/infrastructure-features: '[]'
+    
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "maistra.io/v2",
+          "kind": "ServiceMeshControlPlane",
+          "metadata": {
+            "name": "basic",
+            "namespace": "control-plane-namespace"
+          },
+          "spec": {
+            "version": "v2.1",
```

Comment:
This needs to be updated to v2.3 in `build/manifest-templates/clusterserviceversion.yaml`. We forgot to update it in 2.2.

Reply:
Done